argocd/assets/argocd.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: argocd
+---
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: argocd
+  namespace: argocd
+spec:
+  name: argocd
+  chart: argocd
```

Comment:
Is there such a chart?

```bash
~$ helm repo add argo https://argoproj.github.io/argo-helm

~ $ helm search repo argo
NAME                      	CHART VERSION	APP VERSION  	DESCRIPTION
argo/argo                 	1.0.0        	v2.12.5      	A Helm chart for Argo Workflows
argo/argo-cd              	7.5.2        	v2.12.3      	A Helm chart for Argo CD, a declarative, GitOps...
argo/argo-ci              	1.0.0        	v1.0.0-alpha2	A Helm chart for Argo-CI
argo/argo-events          	2.4.7        	v1.9.2       	A Helm chart for Argo Events, the event-driven ...
argo/argo-lite            	0.1.0        	             	Lighweight workflow engine for Kubernetes
argo/argo-rollouts        	2.37.6       	v1.7.2       	A Helm chart for Argo Rollouts
argo/argo-workflows       	0.42.1       	v3.5.10      	A Helm chart for Argo Workflows
argo/argocd-applicationset	1.12.1       	v0.4.1       	A Helm chart for installing ArgoCD ApplicationSet
argo/argocd-apps          	2.0.0        	             	A Helm chart for managing additional Argo CD Ap...
argo/argocd-image-updater 	0.11.0       	v0.14.0      	A Helm chart for Argo CD Image Updater, a tool ...
argo/argocd-notifications 	1.8.1        	v1.2.1       	A Helm chart for ArgoCD notifications, an add-o...

```

Reply:
A yaml like this seems to work (if this is the helm chart that we intended to install, I'm no sure):

```
spec:
  name: argocd
  chart: argo-cd
  repo: https://argoproj.github.io/argo-helm
  valuesContent: |-
    {}
  version: "7.5.2"
```